src/Admin/PageAdmin.php

```diff
@@ -400,6 +400,13 @@ protected function configureFormFields(FormMapper $formMapper)
                             'siteId' => $this->getSubject() ? $this->getSubject()->getSite()->getId() : null,
                         ],
                     ])
+                ->add('targetRedirectType', method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+                    ? 'Symfony\Component\Form\Extension\Core\Type\ChoiceType' : 'choice', [
```

Comment:
in fact, do not do this if. Add directly Choice::type

Reply:
Sorry because it was asked otherwise on a previous comment, but since the start of this PR we changed our deps an we no longer support SF 2.3, the min is 2.8 and you can use FQCN directly on the formTypes

Reply:
Will do, awesome! With the drop of 2.3 I can also use the constants for the status code.